base/shell/explorer/traywnd.cpp

```diff
@@ -3157,7 +2950,21 @@ class CTrayWindow :
 
     LRESULT OnNcLButtonUp(UINT uMsg, WPARAM wParam, LPARAM lParam, BOOL& bHandled)
     {
-        CheckShowDesktopButtonClick(lParam, bHandled);
+        if (m_pShowDesktopButton && m_pShowDesktopButton->m_bPressed) // Did you click the button?
+        {
+            m_pShowDesktopButton->Click();
+            m_pShowDesktopButton->OnLButtonUp(WM_LBUTTONUP, 0, 0, bHandled);
+            bHandled = TRUE;
+            //return TRUE;
```

Comment:
Why commented?

Reply:
I forgor :skull: to remove that...will fix  :+1: